thunderbird/MozillaThunderbird.spec

```diff
@@ -341,18 +330,27 @@ echo ""
 cat << EOF
 %else
 %limit_build -m 2000
+export MOZ_DEBUG_FLAGS="-pipe"
 cat << EOF > $MOZCONFIG
-%endif
```

Comment:
I'm not sure why this %endif was removed here. Everything below should be part of the actual build _and_ `only_print_mozconfig`.
I think line 414 has the "new" endif, which should probably be removed if this endif is going to come back

Reply:
%if 0%{?suse_version} > 1320
%ifarch i586
mk_add_options MOZ_MAKE_FLAGS=-j1
%else
mk_add_options MOZ_MAKE_FLAGS=%{?jobs:-j%jobs}
%endif
%endif

looks correct?

Reply:
indeed I seem to have moved the endif to 414 which is actually wrong. Reverted now